src/partials/layout.html

```diff
@@ -221,5 +221,54 @@
       });
     </script>
     <script async src="https://cdn.announcekit.app/widget-v2.js"></script>
+    <script>
+      filterSelection("all")
+      function filterSelection(c) {
+        var x, i;
```

Comment:
It's cleaner to declare the variables together with assignment, like `const x = document.getElementsByClassName("filterDiv");`. Also, `x` is not particularly descriptive, could be something like `integrationCards` :)